src/libmagickwand.jl

```diff
@@ -19,39 +19,131 @@ export MagickWand,
+    MagickGetException[]               = func(:MagickGetException)
+    PixelGetException[]                = func(:PixelGetException)
+    MagickExportImagePixels[]          = func(:MagickExportImagePixels)
+    MagickImportImagePixels[]          = func(:MagickImportImagePixels)
+    MagickConstituteImage[]            = func(:MagickConstituteImage)
+    MagickSetImageDepth[]              = func(:MagickSetImageDepth)
+    MagickGetImagesBlob[]              = func(:MagickGetImagesBlob)
+    MagickPingImage[]                  = func(:MagickPingImage)
+    MagickReadImage[]                  = func(:MagickReadImage)
+    MagickReadImageFile[]              = func(:MagickReadImageFile)
+    MagickReadImageBlob[]              = func(:MagickReadImageBlob)
+    MagickWriteImages[]                = func(:MagickWriteImages)
+    MagickWriteImagesFile[]            = func(:MagickWriteImagesFile)
+    MagickGetImageHeight[]             = func(:MagickGetImageHeight)
+    MagickGetImageWidth[]              = func(:MagickGetImageWidth)
+    MagickGetNumberImages[]            = func(:MagickGetNumberImages)
+    MagickNextImage[]                  = func(:MagickNextImage)
+    MagickResetIterator[]              = func(:MagickResetIterator)
+    MagickNewImage[]                   = func(:MagickNewImage)
+    MagickGetImageAlphaChannel[]       = func(:MagickGetImageAlphaChannel)
+    MagickGetImageProperties[]         = func(:MagickGetImageProperties)
+    MagickGetImageProperty[]           = func(:MagickGetImageProperty)
+    MagickGetImageColors[]             = func(:MagickGetImageColors)
+    MagickGetImageType[]               = func(:MagickGetImageType)
+    MagickSetImageType[]               = func(:MagickSetImageType)
+    MagickGetImageColorspace[]         = func(:MagickGetImageColorspace)
+    MagickSetImageColorspace[]         = func(:MagickSetImageColorspace)
+    MagickSetImageCompression[]        = func(:MagickSetImageCompression)
+    MagickSetImageCompressionQuality[] = func(:MagickSetImageCompressionQuality)
+    MagickGetImageTicksPerSecond[]     = func(:MagickGetImageTicksPerSecond)
+    MagickGetImageDelay[]              = func(:MagickGetImageDelay)
+    MagickSetImageDelay[]              = func(:MagickSetImageDelay)
+    MagickSetImageFormat[]             = func(:MagickSetImageFormat)
+    MagickGetImageDepth[]              = func(:MagickGetImageDepth)
+    MagickGetImageChannelDepth[]       = func(:MagickGetImageChannelDepth)
+    PixelSetColor[]                    = func(:PixelSetColor)
+    MagickRelinquishMemory[]           = func(:MagickRelinquishMemory)
+    MagickQueryConfigureOptions[]      = func(:MagickQueryConfigureOptions)
+    MagickQueryConfigureOption[]       = func(:MagickQueryConfigureOption)
+
+    ccall((:MagickWandGenesis, libwand), Void, ())
+
+    libversion_string = queryoption("LIB_VERSION_NUMBER")
+    global libversion = VersionNumber(join(split(libversion_string, ',')[1:3], '.'))
```

Comment:
Would it be better to use the`Ref{VersionNumber}` version here? `__init__()` isn't called during compilation, but `libversion` is used in a precompiled function (https://github.com/musm/ImageMagick.jl/blob/b1178deb283ee2e5ec1e449ddb4b80a7c67e728e/src/ImageMagick.jl#L155).